<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1177</width>
                <height>681</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                        <widget class="QPushButton" name="ROT13Button">
                            <property name="minimumSize">
                                <size>
                                    <width>120</width>
                                    <height>45</height>
                                </size>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>120</width>
                                    <height>45</height>
                                </size>
                            </property>
                            <property name="text">
                                <string>ROT13</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="ROT47Button">
                            <property name="minimumSize">
                                <size>
                                    <width>120</width>
                                    <height>45</height>
                                </size>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>120</width>
                                    <height>45</height>
                                </size>
                            </property>
                            <property name="text">
                                <string>ROT47</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <spacer name="horizontalSpacer">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QPushButton" name="ROTEncodeButton">
                            <property name="minimumSize">
                                <size>
                                    <width>120</width>
                                    <height>45</height>
                                </size>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>120</width>
                                    <height>45</height>
                                </size>
                            </property>
                            <property name="text">
                                <string>计算!</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item row="1" column="0">
                <widget class="QTextEdit" name="ROTTextBox"/>
            </item>
            <item row="1" column="1">
                <widget class="QTextEdit" name="ROTCipherBox"/>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
